src/main/resources/swagger/api-docs.yaml

```diff
@@ -1425,6 +1425,49 @@ paths:
           description: Method Configuration failed to resolve input expressions with the supplied Entity
         500:
           description: Internal Error
+  '/api/libraries':
+    get:
+      tags:
+        - Library
+      operationId: FindDocuments
+      summary: |
+        lists published library datasets
+      produces:
+        - text/plain
+        - application/json
+      parameters:
+        - in: query
+          description: which result to start at (for pagination)
+          name: from
+          type: integer
+          required: false
+        - in: query
+          description: how many results to return
+          name: size
+          type: integer
+          required: false
```

Comment:
for `from` and `size` parameters, specify a `default`: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameterObject